ui/app/models/replication-attributes.js

```diff
@@ -28,6 +28,16 @@ export default Fragment.extend({
       ? 'bootstrapping'
       : (this.get('isSecondary') && 'secondary') || (this.get('isPrimary') && 'primary');
   }),
+  modeForHeader: computed('mode', function() {
+    const mode = this.get('mode');
+    if (mode === 'bootstrapping') {
```

Comment:
wouldn't this case be handled below where it returns the value of `mode` if it's not falsy?

Reply:
yes, yes it would.  Conditionals hurt my brain, thanks for helping it out here.